controller/planner_backlog.go

```diff
@@ -93,10 +93,11 @@ func generateBacklogExpression(ctx context.Context, db application.DB, spaceID u
 
 		// Get the list of work item types that derive of PlannerItem in the space
 		var expWits criteria.Expression
-		wits, err := appl.WorkItemTypes().ListPlannerItems(ctx, spaceID)
+		witsOrig, err := appl.WorkItemTypes().ListPlannerItems(ctx, spaceID)
 		if err != nil {
 			return errs.Wrap(err, "unable to fetch work item types that derive from planner item")
 		}
+		wits := stripBaseWorkItemTypes(witsOrig)
```

Comment:
I think the previous query already excludes the SystemPlannerItem bc doesn't derive of himself :). So I am not sure if we need this.